### DISCUSSION TITLE
模块功能不可用, 因为内核不支持OverlayFS

### DISCUSSION THREAD

**Discussion by lazycat55** [bmax121/APatch#415] [Q&A] [answered]
OnePlus A5000, 安卓8.1.0 系统模块提示 模块功能不可用, 因为内核不支持OverlayFS, 该如何解决?
kernelPatch 版本 0.10.5
APatch 版本 10657
内核版本 4.4.78-perf+
系统版本 8.1.0(API 27)

**Comment by pomelohan** [comment, marked as answer]
自行编译一个带 OverlayFS 支持的内核，或者用别人维护的 OverlayFS supported 内核
